src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -235,9 +236,19 @@ public void canCreateUpdateAndDeleteFares() throws IOException, SQLException, In
         assertThat(createdFare.fare_id, equalTo(fareId));
         assertThat(createdFare.fare_rules[0].fare_id, equalTo(fareId));
 
+        // Ensure transfers value is null.
+        ResultSet resultSet = getResultSetForId(createdFare.id, Table.FARE_ATTRIBUTES, "transfers");
+        while (resultSet.next()) {
+            // We must use getObject because getInt will always return 0 for a null value.
+            Object value = resultSet.getObject(1);
+            LOG.info("fare#transfers {}", value);
+            assertThat(value, Matchers.nullValue());
```

Comment:
Can you also add some other assertions for other fields to make this a more comprehensive test of whether the values are stored correctly in the database?